src/main/java/com/soboleoko/medicalclinic/model/Patient.java

```diff
@@ -0,0 +1,119 @@
+
+package com.soboleoko.medicalclinic.model;
+
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+
+@RequiredArgsConstructor
```

Comment:
Dodajesz konstruktor do pol wymagalnyhc (finalne i constrainty takie jak not null np. ) a w klasie nie masz zadnego pola finalnego. 
Tutaj uzyjmy @AllArgsConstructor